Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN pip install --upgrade conan
 # configure the build with symbols turned on so that crashes can be triaged
 WORKDIR /usr/local/src/valhalla/build
 RUN cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_C_COMPILER=gcc
-RUN make all -j${CONCURRENCY}
+RUN make all -j${CONCURRENCY:-$(nproc)}
```

Comment:
i was wondering why you didnt do it like this in the previous one but couldnt be bothered to check docker to see what would happen